src/hw-2/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { User } from './types';
+
+function sortListByLogin(a: User, b: User) {
+    const textA = a.login.toLowerCase();
+    const textB = b.login.toLowerCase();
+
+    if (textA > textB) {
+        return 1;
+    } else if (textA < textB) {
+        return -1;
+    } else {
+        return 0;
+    }
+}
+
+export function getAutoSuggestUsers(loginSubstring: string, limit: number, userList: User[]): User[] {
+    // userList should be sorted if items number in userList are more than one item
+    const newStore = userList.length > 1
```

Comment:
if there are no users in the list -> sort and filter callbacks won't run anyway, no sense to check the array's length.

Reply:
I've added this veriication in case if there's one user in the store: it doesn't make sense to sort/filter the array.

Reply:
I've added this verifcation in case if there's one user in the store: it doesn't make sense to sort/filter the array.